content/help/security-vulnerabilities/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Istio Security Vulnerability Disclosure Information
+description: Responsible disclosure for Istio security vulnerabilities
+weight: 35
+aliases:
+    - /security-vulnerabilities.html
+    - /security-vulnerabilities/index.html
+---
+## How to report a security vulnerability?
+
+We’re very grateful for security researchers and users that report Istio security vulnerabilities to us. We will thoroughly investigated every report. 
+
+To make a report, please email the private [istio-security-vulnerabilities@google.com](mailto:istio-security-vulnerabilities@google.com) list with the vulnerability details and the details expected for [Istio bug reports](https://istio.io/help/bugs/).
+
+### When should I report a security vulnerability?
+
+- You think you discovered a potential security vulnerability in Istio
+- You are unsure whether/how a vulnerability affects Istio
+- You think you discovered a vulnerability in another project that Istio depends on (e.g., Envoy, docker, Kubernetes)
+
+### When should I NOT report a security vulnerability?
+
+- You need help tuning Istio components for security
+- You need help applying security related updates
+- Your issue is not security related
+
+## Security vulnerability response
+
+Each report is acknowledged and analyzed by Istio security team members within 3 working days. 
+
+Any vulnerability information shared with Istio security team stays within Istio project and will not be disseminated to other projects unless it is necessary to get the issue fixed.
```

Comment:
Run-on sentence and verb tense congruence:
Any vulnerability information you share with the Istio security team stays within the Istio project. We don't disseminate the information to other projects. We only share the information as needed to fix the issue.

Reply:
Done.